src/components/Video/Video.module.scss

```diff
@@ -14,6 +14,7 @@
     width: 100%;
     height: 100% !important;
     padding: 0 !important;
+    background-color: white;
```

Comment:
Was this the actual fix, or is it the changes to aspect ratio below?

If not, aspectRatio can be either `X/Y` or a decimal. Therefore original format was fine fyi.

Reply:
Might be better to use the page --background-color variable

Reply:
It was both from what I checked. I thought giving the aspect ratio as `x/y` would let the browser decide on the precision it needs.
Besides this, I also completely removed the aspect ratio on the video div. Right now it's only on the placeholder image. This reduced the black bar, but not completely eliminate it. 
That's why on top of that, I made the background white as well.